nss-cohort-36/bangazon-client-virtual-magicians-1
# Sp user auth

# Description
**Register.py** is functional where now a user can visit `localhost:3000/register` and create a new account.
Closes https://github.com/nss-cohort-36/bangazon-api-virtual_magicians/issues/51
## Type of change

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-user-auth`
4. Go to **http://localhost:3000/register** and create a new account. Make sure it posts to your database (TablePlus) and redirects you to the home page
5. **Make sure once you are logged in, you can now view the full NavBar including Sell and Shopping Cart**

# Checklist:
